c2corg_ui/templates/helpers/view.html

```diff
@@ -74,6 +74,15 @@
   ${get_attr(version, 'comment', md=False)  if version['comment'] else '' | n}
 </%def>
 
+<%def name="show_article_title(locale, is_tab_title=False)">\
```

Comment:
I am not sure. Show_title cause errors (TypeError: unsupported operand type(s) for +: 'dict' and 'str')... In areas you used show_route_title. Maybe it is better to use it here too and it will be easier to find and change it later with other parts named by "route"...